src/libui_sdl/main.cpp

```diff
@@ -1512,6 +1516,41 @@ void OnAreaResize(uiAreaHandler* handler, uiArea* area, int width, int height)
 }
 
 
+void LoadSlot2Cart()
```

Comment:
I would move the cart selection into the core. Otherwise this code needs to be duplicated for each frontend